src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -302,6 +302,9 @@
   <StringProperty Name="TargetPlatformVersion"
                   Visible="false" />
 
+  <StringProperty Name="TargetPlatformMinVersion"
```

Comment:
This is unneeded. 

We might want to consider exposing this property inside VS to other components, but this is the correct place to do it. I'll hold off exposing until there's an ask.